src/data/roadmapData.json

```diff
@@ -144,12 +144,22 @@
     {
       "Terminal": [
         {
-          "name": "placeholder",
+          "name": "Basic terminal commands",
           "data": [
             {
-              "description": "## description1 \n * list1 \n * list2 \n",
-              "url": "http://google.com",
-              "urlLabel": "some urlLabel"
+              "description": "Here's list of basic terminal commands used in everyday work",
+              "url": "https://www.techrepublic.com/article/16-terminal-commands-every-user-should-know/",
+              "urlLabel": "Terminal tech republic"
+            },
+            {
+              "description": "Netguru wiki - terminal instructions. Here you can find useful commands gathered from Netgurals",
+              "url": "https://netguru.atlassian.net/wiki/spaces/QA/pages/270533008/Terminal",
+              "urlLabel": "Netguru page for using terminal"
+            },
+            {
+              "description": "Variables",
```

Comment:
Better to specify it a bit more like "How to use environment and Shell variables"

Reply:
Done